op-e2e/actions/upgrades/interop_fork_test.go

```diff
@@ -0,0 +1,144 @@
+package upgrades
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/interop/dsl"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/upgrades/utils"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/event"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInteropUpgrade(gt *testing.T) {
```

Comment:
I don't think it needs to block this PR, but previously we had migrated some hardfork action tests to the "proofs" folder, meaning that we can run op-program and kona on them (although I don't spot any action tests covering the Isthmus upgrade transactions). 

Let's discuss on Discord on whether we want to ticket such a migration for this test.

Reply:
The old proofs action tests won't work for the interop upgrade, but it's possible the new proofs action tests could be made to work. Regardless it would be a new pattern and we are a bit time constrained so this tests only op-node/op-geth derivation.

Reply:
TODO in follow-up